tests/testdata/config/virtual-service-all.yaml

```diff
@@ -6,8 +6,7 @@ spec:
   hosts:
   - service3.default.svc.cluster.local
   - uk.bookinfo.com
-  # Bug: dup not rejected
-  #- eu1.bookinfo.com
+  - eu1.bookinfo.com
```

Comment:
the same host is defined in rule-route-via-egressgateway.yaml

Reply:
Ah thats why its failing..